aten/src/ATen/SparseTensorUtils.h

```diff
@@ -128,4 +129,37 @@ inline LongTensor flatten_indices_by_dims(const LongTensor& indices, const IntAr
   return new_indices;
 }
 
+inline LongTensor coo_to_csr(const int64_t* indices, int64_t dim, int64_t nnz) {
+  /* 
```

Comment:
Let's stop using the type "LongTensor" and replace it with Tensor like in the rest of the codebase. (Same for the other types defined alongside it.)

Can this function be defined in a .cpp file and just declared here?

Reply:
Ok. Yes it can be defined in a .cpp file.